aptos-move/framework/aptos-framework/sources/staking_proxy.spec.move

```diff
@@ -6,9 +6,12 @@ spec aptos_framework::staking_proxy {
 
     /// Aborts if conditions of SetStakePoolOperator are not met
     spec set_operator(owner: &signer, old_operator: address, new_operator: address) {
+        pragma verify = false;
```

Comment:
If it can be proved with `verify_duration_estimate = 360`, please remove it

Reply:
There is a vector_ref loop in `set_vesting_contract_operator` and `set_staking_contract_operator`, so we used the pragma = false.